pulp_python/app/serializers.py

```diff
@@ -334,6 +345,13 @@ class PythonPublicationSerializer(core_serializers.PublicationSerializer):
     A Serializer for PythonPublication.
     """
 
+    python_distributions = core_serializers.DetailRelatedField(
```

Comment:
I think we want to establish a convention and call this field `distributions` in all plugins (@bmbouter?) (where it is applicable, i.e. there is a Publication).  For example, see the file plugin: https://github.com/pulp/pulp_file/blob/88f5bd302d1c98b7b34b474fe9bf8d6d9e7cce0c/pulp_file/app/serializers.py#L84